ChangeLog

```diff
@@ -1,3 +1,5 @@
+	* remove deprecated functions from python examples
+	* add piece index and block index python bindings
```

Comment:
I believe there are already changelog entries for deprecating the functions. I don't think updating the python binding needs its own entry. The same thing applies to the piece_index_t type. I believe it's mentioned in a separate entry